modules/course_management/util.py

```diff
@@ -1,35 +1,40 @@
 from typing import Union, overload
 import discord
 import config
-from utils.utils import get_id
+from utils.utils import get_discord_obj, get_id
 
 
-@overload
-def is_course(channel_id: int, /) -> bool:
-	...
-
-
-@overload
-def is_course(channel: discord.TextChannel, /) -> bool:
-	...
-
-
-def is_course(arg1: Union[int, discord.TextChannel]) -> bool:
-	channel = (
-		discord.utils.get(config.guild().text_channels, id=arg1)
-		if isinstance(arg1, int)
-		else arg1
-	)
+def is_course(channel: discord.TextChannel) -> bool:
 	return channel is not None and channel.category_id in {
 		get_id("ACTIVE_COURSES_CATEGORY"),
 		get_id("INACTIVE_COURSES_CATEGORY"),
```

Comment:
Could these strings use the constants from before? May need to move some things around to do it